src/Pos/Ssc/Arbitrary.hs

```diff
@@ -2,12 +2,13 @@ module Pos.Ssc.Arbitrary
        ( SscPayloadDependsOnSlot (..)
        ) where
 
-import           Test.QuickCheck     (Arbitrary (..), Gen)
 import           Universum
 
-import           Pos.Ssc.Class.Types (Ssc (..))
-import           Pos.Ssc.NistBeacon  (SscNistBeacon)
-import           Pos.Types           (SlotId)
+import           Test.QuickCheck          (Arbitrary (..), Gen)
+
+import           Pos.Ssc.Class.Types      (Ssc (..))
+import           Pos.Ssc.NistBeacon       (SscNistBeacon)
+import           Pos.Types                (SlotId)
```

Comment:
I think this module `Pos.Ssc.Arbitrary` should go to `cardano-sl-ssc`. It imports two modules from `cardano-sl`: `Pos.Ssc.NistBeacon` and `Pos.Types`. The latter can be changed to `Pos.Core`. The former can be moved to `cardano-sl-ssc`, it only imports `Pos.Types` which can be changed to `Pos.Core` as well.